Tomek and OHaraRudyIso as well as smaller changes

Jan-ErikDuhme opened the pull request:
- Changed PCa Multiplier in OHaraRudy mid ev-File from 1.8 to 2.5
- Implemented Tomek ionic model
- Implemented OHaraRudy model with Isoprenaline
- Added option to specify inital values of the Land17 tension model in the xml file

Commit:
build Ordner hinzugefügt

Commit:
XCode Projekt hinzugefügt

Commit:
Kleine Änderungen XCode Projekt

Commit:
XCode Projekt Änderung

Commit:
Befehl fürs inititlaisieren der Werte in CBTensionModelLand17.cpp hinzugefügt

Commit:
t und delta_t müssen keine anderen Initialwerte haben

Commit:
Neu hinzugefügte BVariablen für die Initialwerte in CBTensionModelLand17.h initialisiert

Commit:
Lambda und delta_lambda entfernt

Commit:
Added O'Hara-Rudy model with Isoprenaline from previously written code

Commit:
Added Tomek ionic model using existing code

Commit:
removed build folders from git

Commit:
Merge branch 'Implementing-O'HaraRudy-Isoprenaline' into Land_initialValues

Merging code with O'HaraRudyIso and Tomek ionic model into the implementation of the initial values for the Land cardiac contraction model

Commit:
solved error by linking files instead of using alias

Commit:
fractions of PKA phosphorylation for OHaraRudyIso can now be specified in the ev-Files

Commit:
changed PCa_Multiplier of OHaraRudy_mid.ev from 1.8 to 2.5 as speciefied in original paper

tobias-gerach commented on electrophysiology/src/CellModel/OHaraRudyIsoParameters.cpp:
```diff
@@ -0,0 +1,384 @@
+  P[VT_RToverF].value = (P[VT_R].value * P[VT_T].value) / P[VT_F].value;
+    
+  // PKA fraction
+  //P[VT_ICaL_P_frac].value  = 0.987823373540079;
```
Please remove unused commented parameters or add a comment what they are/why you want to keep them here

tobias-gerach reviewed (changes requested):
Thanks for the changes and the implementation of new models!

I will accept the PR if you do some small changes.

- Replace the file header in the Tomek model files with the standard file header found in all other models
- Give some context for what is different between the implementation of OHaraRudy and OHaraRudyIso
- Give some context if the Tomek model is in its original state from the publication and what changes/additions you made if you made any
- Remove the unused commented parameters in electrophysiology/src/CellModel/OHaraRudyIsoParameters.cpp or give context why you want to keep them

A short comment here about this stuff is fine for me.

Commit:
changed header for Tomek ionic model and removed uncommented code from OHaraRudyIsoParameters.cpp

Jan-ErikDuhme commented:
I have updated the headers for the Tomek model and removed the commented lines of code. Differences of the Tomek model compared to the original paper: equations for SAC, as well as TRPN coupling with the Land tension model. Both have been adopted directly from the OHaraRudy model.

The OHaraRudyIso model adds PKA phosphorylation to multiple different currents as specified in the paper by Heijman et al. 2011 (https://doi.org/10.1016/j.yjmcc.2011.02.007). Only adaptations directly to the OHaraRudy model were implemented. The fractions of PKA phosphorylation can be specified in the corresponding ev-File (...P_frac).

Let me know if you need any more information.
